include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -2309,6 +2332,11 @@ namespace internal
                        1. :
                        my_data.descriptor[0].quadrature.weight(q));
 
+#ifdef DEBUG
+                  for (unsigned int v = 0; v < n_lanes_d; ++v)
+                    Assert(JxW[v] > 0.0, ExcInternalError());
```

Comment:
I wondered whether this exc message is clear, i.e. points to an invalid Jacobian determinant on faces? One might argue that JxW is basic FE language, so this is probably a matter of taste.